crates/brainfuck_prover/src/components/memory/table.rs

```diff
@@ -199,8 +199,7 @@ impl MemoryTable {
         // TODO: Confirm that the log_size used for evaluation on Circle domain is the log_size of
         // the table plus the SIMD lanes
         let log_size = log_n_rows + LOG_N_LANES;
-        let mut trace: Vec<BaseColumn> =
-            (0..N_COLS_MEMORY_TABLE).map(|_| BaseColumn::zeros(1 << log_size)).collect();
+        let mut trace = vec![BaseColumn::zeros(1 << log_size); N_COLS_MEMORY_TABLE];
```

Comment:
With the refactor to enum:

```suggestion
        let mut trace = vec![BaseColumn::zeros(1 << log_size); MemoryColumn::count()];
```